test/ofex_test.exs

```diff
@@ -40,13 +40,13 @@ defmodule OfexTest do
   end
 
   test "can parse QFX data" do
-    ofx_raw = File.read!("test/fixtures/bank_account.qfx")
-    {:ok, %{accounts: [_bank_account]}} = Ofex.parse(ofx_raw)
+    ofx_raw = File.read!("test/fixtures/multiple_accounts.ofx")
+    {:ok, %{accounts: [_bank_account1, _bank_account2, _cc_account]}} = Ofex.parse(ofx_raw)
   end
 
   test "can parse! QFX data" do
-    ofx_raw = File.read!("test/fixtures/bank_account.qfx")
-    %{accounts: [_bank_account]} = Ofex.parse!(ofx_raw)
+    ofx_raw = File.read!("test/fixtures/multiple_accounts.ofx")
+    %{accounts: [_bank_account1, _bank_account2, _cc_account]} = Ofex.parse!(ofx_raw)
```

Comment:
Sorry, I probably should have been more clear. We shouldn't adjust these tests because they are mainly asserting that it can handle specific QFX format files.

Instead, can you add new test cases in `Ofex.BankAccountTest` and `Ofex.CreditCardAccountTest` for testing parsing multiple accounts in a file? Asserting that account details and a couple transactions in each

Reply:
Hello Jon, 
I have updated the test cases for parsing multiple accounts in BankAccounts and CreditCards.